src/pybind/mgr/dashboard/services/rgw_client.py

```diff
@@ -206,13 +207,23 @@ def _get_user_id(self, admin_path, request=None):
         return response['data']['user_id']
 
     @RestClient.api_get('/{admin_path}/metadata/user', resp_structure='[+]')
-    def _is_system_user(self, admin_path, request=None):
+    def _user_exists(self, admin_path, request=None):
```

Comment:
Is `admin_path` set anywhere set to something different than `self.admin_path`?

Reply:
``admin_path`` is configurable via CLI because of 'An admin API request will be done on a URI that starts with the configurable ‘admin’ resource entry point.', see http://docs.ceph.com/docs/master/radosgw/adminops/.